Test/Test.java

```diff
@@ -0,0 +1,57 @@
+//getting familiar with data types, variables, conversion and casting 
+
+public class Test{
+
+	public static void main(String[] args) {
+		//variable name rules-
+		int num;
+		int _number;
+		int $num;
+		int num123;
+		int numOfCars;
+		// wrong name - int 12per;
+		
+		//data types
+
+		byte b = 8;
+		short sh = 11;
+		int a = 10;
+		long l = 18273647888890009L;
+		float  f = 20.3f;
+		double c = 3.154637;
+		float h = Float.MAX_VALUE;
+		String s = "\\Neeraja Sekhar\\";
+		String s1= "\'Neeraja Sekhar\'";
```

Comment:
For single quote `'` you don't have to use negating wild character `\`. You can just write System.out.println(" '  YOUR_NAME  ' ") and will get compiled fine.